code/clue.js

```diff
@@ -1,59 +1,353 @@
+  color: 'white',
+  description: '',
+  age: 75,
+  image: 'assets/white.png',
+  occupation: 'retired',
+  favouriteWeapon: "candlestick"
+}
+
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 10,
+  id: 'rope'
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 6,
+  id: 'knife'
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 8,
+  id: 'candlestick'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 3,
+  id: 'dumbbell'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const poison = {
+  name: 'Poison',
+  weight: 2,
+  id: 'poison'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 8,
+  id: 'axe'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 5,
+  id: 'bat'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 5,
+  id: 'trophy'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 4,
+  id: 'pistol'
+}
+
+
+//Grouped into arrays
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
 ]
 
-const weapons = []
+let mystery = {
+  killer: 'who',
+  weapon: 'what',
+  room: 'where'
+}
+
+let killerWeapons = Array(8).fill('');
+let moreWeapons = []; 
 
-const rooms = []
+// FUNCTIONS
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
 
-// This function will be invoked when you click on the killer card.
-const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
+const shuffleFavouriteWeapon = () => {
+  suspects.forEach(element => {
+    element.favouriteWeapon = randomSelector(weapons)
+  });
+  
+}
+  
+const showKillerLoader = () => {
+  document.getElementById('killerLoader').style.display = 'flex';
+  document.getElementById('killerHeadline').style.display = 'none';
+  document.getElementById('killerTitle').style.display = 'none';
+  document.getElementById('killerContent').style.display = 'none';
+  document.getElementById('killerDecoration').style.display = 'flex'
+  document.getElementById('killerCard').style.background = 'black'
+  document.getElementById('killerDecoration').style.border = '8px solid rgb(129, 164, 175)'
```

Comment:
Remember to be consistent with the color code. Do you want to use HEX or rgb or words? I see that you mostly use HEX so maybe use HEX for your whole site.

Reply:
Agreed!